File: Nian_Karl/SoftwareConstruction/src/org/uva/qls/ast/widget/Checkbox.java
package org.uva.qls.ast.widget;

import org.uva.utility.CodePosition;

public class Checkbox extends WidgetType {

	public Checkbox(CodePosition position) {
		super(position);
	}
	
	@Override
	public String toString() {
		return "checkbox";
	}
}

File: CarlaErik/UvA.SoftCon.Questionnaire/QLS/AST/Model/widget/Checkbox.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UvA.SoftCon.Questionnaire.Utilities.AST;

namespace UvA.SoftCon.Questionnaire.QLS.AST.Model.widget
{
    public class Checkbox : QLSNode
    {
        public bool IsComputed
        {
            get;
            private set;
        }

    }
}
